airbyte-cdk/bulk/core/base/build.gradle

```diff
@@ -13,29 +13,29 @@ dependencies {
         exclude group: 'org.apache.commons'
         exclude group: 'commons-io'
     }
-    api 'io.github.oshai:kotlin-logging-jvm:5.1.0'
+    api 'io.github.oshai:kotlin-logging-jvm:7.0.0'
     api 'io.micronaut:micronaut-runtime'
-    api 'org.apache.sshd:sshd-mina:2.12.1'
+    api 'org.apache.sshd:sshd-mina:2.13.2'
     api 'org.jetbrains.kotlinx:kotlinx-coroutines-core'
 
-    implementation 'com.datadoghq:dd-trace-api:1.28.0'
-    implementation 'com.datadoghq:dd-trace-ot:1.28.0'
+    implementation 'com.datadoghq:dd-trace-api:1.39.0'
+    implementation 'com.datadoghq:dd-trace-ot:1.39.0'
     implementation 'com.fasterxml.jackson.module:jackson-module-kotlin'
     implementation 'com.fasterxml.jackson.module:jackson-module-afterburner'
     implementation 'info.picocli:picocli:4.7.6'
     implementation 'io.micronaut.picocli:micronaut-picocli:5.5.0'
-    implementation 'jakarta.validation:jakarta.validation-api:3.0.2'
-    implementation 'org.apache.commons:commons-lang3:3.14.0'
+    implementation 'jakarta.validation:jakarta.validation-api:3.1.0'
+    implementation 'org.apache.commons:commons-lang3:3.17.0'
     implementation 'org.apache.logging.log4j:log4j-api'
     implementation 'org.apache.logging.log4j:log4j-core'
     implementation 'org.apache.logging.log4j:log4j-slf4j-impl'
     implementation 'org.apache.logging.log4j:log4j-slf4j2-impl'
-    implementation 'org.apache.logging.log4j:log4j-layout-template-json:2.17.2'
-    implementation 'org.bouncycastle:bcprov-jdk18on:1.77'
+    implementation 'org.apache.logging.log4j:log4j-layout-template-json:2.24.0'
+    implementation 'org.bouncycastle:bcprov-jdk18on:1.78.1'
     implementation 'org.openapi4j:openapi-schema-validator:1.0.7'
 
-    runtimeOnly 'com.google.guava:guava:33.2.0-jre'
-    runtimeOnly 'org.apache.commons:commons-compress:1.26.1'
+    runtimeOnly 'com.google.guava:guava:33.3.0-jre'
```

Comment:
Watch out for these. These, I think but I'm not sure, are here to turn transitive dependencies into runtime-only. There's no point in using Guava in a Kotlin codebase.

I really should have commented their purpose, because now I can't really remember any more!

Reply:
I'll find out what their purpose is and will report back shortly.

Reply:
These two `runtimeOnly` dependencies exist because of the exclusions defined for `airbyte-protocol`. Can you add a comment to this effect please?

In any case bumping their versions is fine.